### DISCUSSION TITLE
Signal missed when playing games on the phone

### DISCUSSION THREAD

**Discussion by UggBugg** [NightscoutFoundation/xDrip#3057] [Q&A]
Hi!
I have xdrip and OOPalgorithm2 on a Samsung S22 Ultra.

When I play a game on my phone, I loose connection to what seems to be the OOPalgorithm2 application. 

I have unrestricted energy usage on the apps and wonder if any other has experienced this problem?

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Enable collector, force foreground, 1 second?

**Reply by UggBugg** [reply]
Hi!
I do have force foreground and 1 second but don't have enable collector. Is that a smart watch feature in xdrip or where can I find that setting?

**Reply by psonnera** [reply]
Sorry, I meant in OOP2 screen, not xDrip+

**Reply by UggBugg** [reply]
Thank you for answering!
I was a bit confused, because I couldn't find enable collector in OOP2. I don't have "use direct intent" marked however. 
I have "use service", "use foreground service" and 1 second in OOP2. 

I have OOP2 version 27da3f5-2023-05-25

**Reply by psonnera** [reply]
This is correct. Did you double check OOP2 is not under optimization in Android settings? It can always run in foreground?

**Reply by UggBugg** [reply]
Yes, it is not optimized or restriced in amy way. It works until I use some apps. For example the games 20 mintues till dawn, Carpet bombing 2 and Pokemon Unite, to mention some. Otherwise it works just fine

**Reply by Navid200** [reply]
Can you look at those game settings to see if they have a setting to keep other apps disabled as long as playing to improve immersion?

If anything disables xDrip from running in the background, xDrip will not be able to get readings.

**Reply by UggBugg** [reply]
I will look into that! It kind of looks that it is OOP2 that doesn't recieve readings but maybe it is because xDrip doesn't send it to OOP2 in the first place? xDrip shows signal missed until I see a received data point in OOP2
